spec/index.bs

```diff
@@ -1187,6 +1187,23 @@ The default is used if a value is not explicitly specified in {{GPUDeviceDescrip
         is {{GPUBufferBindingType/"storage"}}
         or {{GPUBufferBindingType/"read-only-storage"}}.
 
+    <tr><td><dfn>minUniformBufferOffsetAlignment</dfn>
```

Comment:
I wonder if there is a correlation between the uniform and storage offset alignment. Any idea?
Perhaps, we could expose a single limit here instead of two.

Reply:
I don't think so, based on the PR summary which implies some devices have 256 for uniform but something smaller for storage.